Base/res/themes/Default.ini

```diff
@@ -67,6 +67,12 @@ SyntaxControlKeyword=#a42ea2
 SyntaxIdentifier=#000000
 SyntaxPreprocessorStatement=#00a0a0
 SyntaxPreprocessorValue=#a00000
+SyntaxFunction=blue
+SyntaxVariable=black
+SyntaxCustomType=orange
+SyntaxNamespace=orange
+SyntaxMember=red
+SyntaxParameter=#eb6d1e
```

Comment:
Would appreciate it if someone more experienced with themes could come later and improve my visual choices here :)
And of course It would be awesome to have support for these color roles in other themes as well.

Reply:
I'm not an expert, but for me the colors are nice and usable... +1 for themes support in the future :)